source/windows/iocp/iocp_event_loop.c

```diff
@@ -118,10 +124,23 @@ void aws_overlapped_reset(struct aws_overlapped *overlapped) {
     AWS_ZERO_STRUCT(overlapped->overlapped);
 }
 
+struct aws_event_loop_vtable s_iocp_vtable = {
+    .destroy = s_destroy,
+    .run = s_run,
+    .stop = s_stop,
+    .wait_for_stop_completion = s_wait_for_stop_completion,
+    .schedule_task_now = s_schedule_task_now,
+    .schedule_task_future = s_schedule_task_future,
+    .connect_to_io_completion_port = s_connect_to_io_completion_port,
+    .is_on_callers_thread = s_is_event_thread,
+    .unsubscribe_from_io_events = s_unsubscribe_from_io_events,
+};
+
 struct aws_event_loop *aws_event_loop_new_default(struct aws_allocator *alloc, aws_io_clock_fn *clock) {
     assert(alloc);
     assert(clock);
 
+    /* load the windows dlls and dynamically bind our soul to Satan's dark angels. */
```

Comment:
I like that this was an addition on its own. That said, maybe not for public release :)